src/components/Clickable/Clickable.scss

```diff
@@ -1,9 +1,17 @@
 @import '~om';
 
 .Clickable {
+  @include text(body1);
```

Comment:
So you're changing the styles of `Clickable` too? Are you sure this doesn't break anything visually on the website?

Reply:
pretty sure, by specifying this, I am not changing nothing that already was there (I hope so)

Reply:
the <a /> has a different font-family and size